### DISCUSSION TITLE
I need a more detailed guidance for how to run with 7900xtx on ubuntu.

### DISCUSSION THREAD

**Discussion by PennyFranklin** [nod-ai/SHARK#1254] [Q&A]
I tried many times to install this package to the ubuntu, but the best result is just  run on the CPU, for others i even can't run this application. help me plz, I'm a really green hand to ubuntu.

**Comment by powderluv** [comment]
Please check Linux 7900xtx thread and we can help you. Basically you need to install the pro drivers and run in rdna2 mode until the driver is fixed

**Reply by PennyFranklin** [reply]
it seems that the pro driver doesn't support the ubuntu 22 yet
